nautobot_netbox_importer/diffsync/adapters/abstract.py

```diff
@@ -288,6 +288,7 @@ def make_model(self, diffsync_model, data):
             )
         return instance
 
+    # pylint: disable=W0221
```

Comment:
This looks like an incompatibility with newer versions of diffsync, which take a sixth parameter `diff` in the abstract `sync_from` method. Should probably be fixed rather than disabled.

Reply:
I didn't realize that diffsync went up from 1.3.0 to 1.5.0. My bad I didn't look poetry.lock changes good enough.